tests/components/minecraft_server/test_config_flow.py

```diff
@@ -28,11 +30,12 @@ async def test_show_config_form(hass: HomeAssistant) -> None:
     assert result["step_id"] == "user"
 
 
-async def test_lookup_failed(hass: HomeAssistant) -> None:
+async def test_address_validation_failed(hass: HomeAssistant) -> None:
     """Test error in case of a failed connection."""
     with patch(
-        "mcstatus.server.JavaServer.async_lookup",
-        side_effect=ValueError,
+        "homeassistant.components.minecraft_server.api.MinecraftServer.__init__",
```

Comment:
We should patch the library client not our code.

Reply:
Even if I plan to add tests of the api module, where I would then patch the library?

Reply:
Yeah. We should avoid writing the tests to be dependent on integration details. Avoiding that allows refactoring the integration without changing any tests.

Reply:
I plan to add more tests in general soon, will take care of this in one of those future PRs.